### PR TITLE
Feature/loginform

### PR DESCRIPTION
**Pull request by byteconde**

### CODE REVIEW

File: public/ads.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// User -> index.html -> if user in LS -> channel.html
```
Comment (condef5) [inline public/ads.txt]: Please delete this file.

Comment (byteconde) [reply public/ads.txt]: Done

File: public/index.html [hunk 2]
```diff
@@ -6,13 +6,22 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Tutopia Clon</title>
-  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/login.css">
   <link rel="shortcut icon" href="img/favicon.png">
 </head>
 
 <body>
-  <h1>Tutopia</h1>
```
Comment (test0n3) [inline public/index.html]: Was it necessary to delete the main title of this page?

File: public/js/login.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const $form = document.getElementById('loginForm');
+
+function getRgb() {
+  let r = Math.floor(Math.random()*256); // Random between 0-255
```
Comment (test0n3) [inline public/js/login.js]: Just wondering what would happen if the color turns out white or the same color as the background.

File: public/js/main.js [hunk 4]
```diff
@@ -1,24 +1,72 @@
...
+const $listMessage = document.getElementById('listMessage');
+const socket = new WebSocket(`ws://localhost:3000/connection`);
+const messages = [];
 
-const $listChannel = document.getElementById("listChannel");
-const $channelHeader = document.getElementsByClassName("channel-header")[0];
+console.log('🚀');
 
-console.log("🚀");
+function test() {
```
Comment (test0n3) [inline public/js/main.js]: We must define this dummy data function. Fill it with all the data that will be set in localStorage

**Pull request closed without merging**